src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: end;
```

Comment:
According to the code style rules, the property `align-items` should use the value `flex-end` instead of `end` to ensure better compatibility across different browsers.